setup.cfg

```diff
@@ -0,0 +1,31 @@
+[pycodestyle]
+max-line-length = 99
+
+[pylint.MASTER]
+jobs = 3
+max-line-length = 99
+fail-under=9.9
+
+load-plugins =
+  pylint.extensions.bad_builtin,
+  pylint.extensions.broad_try_clause,
+  pylint.extensions.check_elif,
+  pylint.extensions.comparetozero,
+  pylint.extensions.emptystring,
+  pylint.extensions.overlapping_exceptions
+
+[pylint.'MESSAGES CONTROL']
+disable=
+        ; docstring
+        missing-class-docstring,
+        missing-function-docstring,
+        missing-module-docstring,
+        ; black & isort compatibility
+        bad-continuation,
+        line-too-long,
+        ungrouped-imports,
+        wrong-import-order,
+        wrong-import-position,
+        too-many-instance-attributes,
```

Comment:
Can we create a new "section" for those options? 🙂  Right now, it looks like they are here because of black & isort compatibility e.g.:
```cfg
disable=
  ; docstring
  missing-class-docstring,
  missing-function-docstring,
  missing-module-docstring,
  ; black & isort compatibility
  bad-continuation,
  line-too-long,
  ungrouped-imports,
  wrong-import-order,
  wrong-import-position,
  ; library specific
  too-many-instance-attributes,
  too-many-arguments,
  too-many-try-statements
```